declipping/declip-simple.py

```diff
@@ -0,0 +1,111 @@
+"""This function takes one clipped file and declips it. It was created for
```

Comment:
Not quite a function
```suggestion
"""This script takes one clipped file and declips it. It was created for
```